### PR TITLE
Show UTC date in readable format for view result values returned by /sms...

### PR DESCRIPTION
**Pull request by aberke**
... endpoint.

Makes debugging so much easier...

### CODE REVIEW

File: applications/crossbar/src/modules/cb_sms.erl [hunk 1]
```diff
@@ -269,7 +269,14 @@ summary(Context) ->
...
 -spec normalize_view_results(wh_json:object(), wh_json:objects()) -> wh_json:objects().
 normalize_view_results(JObj, Acc) ->
-    [wh_json:get_value(<<"value">>, JObj)|Acc].
+    [normalize_view_result_value(wh_json:get_value(<<"value">>, JObj))|Acc].
+
+-spec normalize_view_result_value(wh_json:object()) -> wh_json:object().
+normalize_view_result_value(JObj) ->
+    CreatedTimestamp = wh_json:get_value(<<"created">>, JObj),
+    {{Year, Month, Day}, {Hour, Min, Secs}} = calendar:gregorian_seconds_to_datetime(CreatedTimestamp),
+    Date = lists:flatten(io_lib:format('~p/~p/~p-~p:~p:~p', [Year, Month, Day, Hour, Min, Secs])),
```
Comment (jamesaimonetti) [inline applications/crossbar/src/modules/cb_sms.erl]: We have a several pretty printers in wh_util:
* pretty_print_datetime/1
* rfc1036/1, rfc1036/2
* iso8601/1

Comment (aberke) [inline applications/crossbar/src/modules/cb_sms.erl]: Great, I'll take a look!

**Comment by aberke** [comment]
I updated the PR @jamesaimonetti

**Comment by aberke** [comment]
Any chance you could add this to 3.19?

**Comment by jamesaimonetti** [comment]
@aberke Done! Will be in the next build

**Comment by aberke** [comment]
thanks

**Pull request merged**